code/game/machinery/computer/_computer.dm

```diff
@@ -30,10 +30,10 @@
 
 /obj/machinery/computer/update_overlays()
 	. = ..()
-	if(machine_stat & NOPOWER)
-		. += "[icon_keyboard]_off"
-		return
-	. += icon_keyboard
+	if(icon_keyboard)
```

Comment:
And make this `if(!icon_keyboard) return`.

Reply:
Same as above, an early return is not intended in the icon_keyboard if